index.html

```diff
@@ -62,5 +62,198 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
+                    </div>
+                    <div class="project-details">
+                        <div class="tech-details">
+                            <h2 class="project-name">Project name goes here</h2>
+                        <div class="tech-stack">
+                            <p>HTML/CSS</p>
+                            <p>Ruby on Rails</p>
+                            <p>JavaScript</p>
+                        </div>
+                        <div class="details-button">
+                        <div>
+                            <p >See this project</p>
+                            <img src="images/Union.svg" alt="union">
+                        </div>
+                        </div>
+                        </div>
+                    </div>
+                </div>
+        </section>
+        <section class="about">
+            <div>
+                <h2 class="headings">About</h2>
+                <div class="content">
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <button class="button download">Get my CV
+                    <div class="icon"></div>
+                </button>
+                </div>
+            </div>
+        </section>
+        <section >
+            <div class="competency">
+                <div class="language grid-item">
+                    <div class="grid-item-head">
+                        <ul>
+                            <li>Languages</li>
+                        </ul>
+                    </div>
+                    <ul>
+                        <li>Javascript(ES5/ES6)</li>
+                        <li>Ruby</li>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                    </ul>
+                </div>
+                <div class="frameworks grid-item">
+                   <div class="grid-item-head">
+                    <ul>
+                        <li>Frameworks</li>
+                    </ul>
+                   </div>
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capibara</li>
+                    </ul>
+                </div>
+                <div class="skills grid-item">
+                   <div class="grid-item-head">
+                    <ul>
+                        <li>Skills</li>
+                    </ul>
+                   </div>
+                    <ul>
+                        <li>Database Management</li>
+                        <li>CLI</li>
+                        <li>API Design</li>
+                        <li>Version Control</li>
+                        <li>Web Development</li>
+                    </ul>
+                </div>
+            </div>
+        </section>   
```

Comment:
As requested by the previous reviewer, please kindly wrap the whole of the `About` section in a single `section` tag.
* A `section` tag represents a generic standalone section of the document. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 🤗

Your code 
```HTML
<section class="about">
    <div>. . . </div>
</section>
<section 
    <div class="competency">
    </div>
</section>
```
Expected
```HTML
<section class="about">
  <div>. . . </div>
  <section 
      <div class="competency">
      </div>
  </section>
</section>
```

Also kindly ensure the content of the "About myself" section is aligned with `Flexbox.`

Let's follow the project core requirements for professional projects, and if we practice from our learning phase, it will be easier in the future at the industry level 👍 . Here is an article on [The importance of understanding software requirements](https://www.freecodecamp.org/news/why-understanding-software-requirements-matter-to-you-as-a-software-engineer/)